conformity/fields/__init__.py

```diff
@@ -17,6 +17,7 @@
     EmailAddress
 )
 from conformity.fields.geo import (  # noqa
+    CountryCodeField,
```

Comment:
We can't import this field here. In fact, we can't even put this field in `conformity.fields.geo` (unless we do something special—I'll point out below). That will make `pycountry` a hard dependency. We're trying to make it a soft dependency (hence why it's an `extra` dependency below). So `CountryCodeField` cannot be imported here at all.